features/admin.applications.v1/api/applications-settings.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2023-2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Reply:
For new files, the year should be 2024. Check other places as well.

Reply:
Fixed with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)